src/components/toolTip/ToolTip.tsx

```diff
@@ -0,0 +1,32 @@
+import clsx from "clsx";
```

Comment:
I see that we now have two Tooltips.

I would prefer just having the _one_. We decided to create a new one because the old implementation had issues. I'd opt to just remove the old one (do make sure to implement it correctly within the implementations).